units/systemd-resolved.service.in

```diff
@@ -37,8 +37,6 @@ ProtectKernelLogs=yes
 ProtectKernelModules=yes
 ProtectKernelTunables=yes
 ProtectSystem=strict
-Restart=always
-RestartSec=0
```

Comment:
OK, so as a compromise, let's keep `Restart=always`, but drop `RestartSec=`?

Reply:
Yeah we should keep the autorestart, it is good to be resilient against unknown glitches